models/glacier/avalanche_c/glacier_avalanche_c_schema.yml

```diff
@@ -0,0 +1,97 @@
+version: 2
+
+models:
+  - name: glacier_avalanche_c_trades
+    meta:
+      blockchain: avalanche_c 
+      sector: dex
+      project: glacier
+      contributors: discochuck 
+    config:
+      tags: ['avalanche_c','glacier','trades','dex']
+    description: >
+        glacier contract trades on avalanche_c
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: avalanche_c 
+          project: glacier
+          version: 1
```

Comment:
Hi @discochuck 
in order for check_dex_seed to work, we may need to add some seeds into the following file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

You can follow the pattern and fill in one or two seed records for testing.

Reply:
i _think_ that should do it.